awx/ui_next/src/components/PaginatedDataList/PaginatedDataList.test.jsx

```diff
@@ -88,16 +51,16 @@ describe('<PaginatedDataList />', () => {
           order_by: 'name',
         }}
         qsConfig={qsConfig}
-      />,
-      { context: { router: { history } } }
+      />, { context: { router: { history } } }
     );
 
     const pagination = wrapper.find('Pagination');
     pagination.prop('onSetPage')(null, 2);
     expect(history.location.search).toEqual('?item.page=2');
     wrapper.update();
     pagination.prop('onSetPage')(null, 1);
-    expect(history.location.search).toEqual('?item.page=1');
+    // since page = 1 is the default, that should be strip out of the search
```

Comment:
I think we should leave the defaults in the urls for two reasons:

1. Users save and bookmark filtered search results so they can come back to them. By eliding some params from the url, we potentially change bookmarked search results if we ever decide to change default page sizes. I've had this happen several times after upgrades, etc.

2. Prettifying the url, omitting defaults, etc. is a nice to have but it's also a significant contributor of complexity in our current implementation. I'm not necessarily against doing it here but in general we should proceed with caution in terms of how much of this sort of thing we commit to supporting.

Reply:
We discussed this offline.  Attempted to see if we could use the same encode qs function everywhere (for api payload and for ui urls), but due to the fact we need to namespace the params to put them url (and not for api), we need to have a qsConfig-aware function.  As such it's only a few extra lines needed to filter those defaults out, we've decided to keep the functionality (and those scenarios are covered in the unit tests).